node/testing/src/cluster/mod.rs

```diff
@@ -388,10 +389,11 @@ impl Cluster {
                 // TODO(binier): record instead of panicing.
                 match res {
                     InvariantResult::Violation(violation) => {
-                        panic!(
```

Comment:
@binier I have to disable it, because it crashes.

Reply:
Just cherry pick this commit and try again. It shouldn't fail.

`6c1cb95941163908880c146823e94c1d1a45f539`

Reply:
Also if an invariant is failing, we can't just disable it. We need to investigate and fix either the invariant or the node. I understand you were working on p2p and invariant that was failing could have been unrelated, but instead of disabling it, please create an issue and ask on the channel for someone to fix it.